config/crd/bases/infrastructure.cluster.x-k8s.io_linodeobjectstoragebuckets.yaml

```diff
@@ -179,6 +179,175 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
```

Comment:
you'd need to update the reference in `config/crd/patches/capicontract_in_linodeobjectstoragebuckets.yaml` to `v1alpha2` as well

Reply:
Hey @tchinmai7 , I have actually made that change in my local version later. Will be pushing it out in a bit.